README

```diff
@@ -261,6 +261,12 @@ Features
       Memory utilization: Memory occupied by running processes, page cache,
       buffer cache and free.
 
+    - mesh
+      Collects various metrics related to Wifi mesh performance (e.g. established
+      peer links, dropped frames, forwarded frames). The statistics are read
+      from the Linux sysfs and require the Linux kernel to be compiled with
```

Comment:
It's read from `debugfs`, not `sysfs` (while distros do mount `debugfs` under `sysfs`, it's a separate fs):
```suggestion
      from the Linux debugfs and require the Linux kernel to be compiled with
```